initialize_database.py

```diff
@@ -0,0 +1,23 @@
+import json
+import sys
+from spinedb_api import import_data, DatabaseMapping, create_new_spine_database
+
+def initialize_database(database_name="new_database.sqlite"):
+
+    #get template JSON
+    with open ('./version/flextool_template_master.json') as json_file:
+        template = json.load(json_file)
+
+
+    new_engine = create_new_spine_database('sqlite:///' +database_name)
+    new_db = DatabaseMapping('sqlite:///' + database_name)
```

Comment:
Not a big issue, but this could be shortened somewhat.

`DatabaseMapping` accepts a `create` parameter that creates the database if it doesn't exist making the call to `create_new_spine_database()` above superfluous. That is, lines 12 and 13 could be replaced by a single line

```python
new_db = DatabaseMapping('sqlite:///' + database_name, create=True)
```